tests/cypress/integration/auth/signup_spec.js

```diff
@@ -25,30 +23,21 @@ describe('Signup', function () {
   //   cy.contains('Login to your account')
   // })
 
-  it('should block registration if policy is not accepted', function () {
-    cy.visit('/register')
+  //it('should block registration if policy is not accepted', function () {
+  //  cy.register(faker.name.firstName(), faker.name.lastName(), faker.internet.password(), faker.internet.email(), false);
+  //  cy.get('.alert').should('exist');
```

Comment:
Is it commented on purpose?

Reply:
I think it's because it's not working in cypress